src/App.css

```diff
@@ -1,4 +1,305 @@
+
+.categorie-stile {
+  margin-bottom: 0;
+  padding-left: 0.5em;
+}
+
+.title {
+  margin: 0;
+  padding-left: 0.5em;
+}
+
+.author {
+  margin-top: 0;
+  color: #4386bf;
+  padding-left: 0.5em;
+}
+
+.book-buttons {
+  width: 20vw;
+}
+
+.book-button {
+  border-radius: 0;
+  border: solid #fff 1px;
+  background-color: #fff;
+  color: #4386bf;
+}
+
+.circle-container {
+  margin: 5px 0;
+  width: 150px;
+  height: 150px;
+  background: #fafafa;
+  border-radius: 50%;
+  border: 1px solid #fafafa;
+  transform: scale(0.6);
+}
+
+.mask .fill {
+  clip: rect(0, 75px, 150px, 0);
+  background-color: #227ded;
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(135deg);
+}
+
+.circle-container .circle .mask,
+.circle-container .circle .fill {
+  width: 150px;
+  height: 150px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-container .circle .mask {
+  clip: rect(0, 150px, 150px, 75px);
+}
+
+.circle-container .inside-circle {
+  width: 122px;
+  height: 122px;
+  border-radius: 50%;
+  background: #fff;
+  line-height: 120px;
+  text-align: center;
+  margin-top: 14px;
+  margin-left: 14px;
+  color: #1e51dc;
+  position: absolute;
+  z-index: 100;
+  font-weight: 700;
+  font-size: 2em;
+}
+
+@keyframes fill {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(135deg);
+  }
+}
+
+.percentage {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  letter-spacing: normal;
+}
+
+.completed {
+  margin: 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  letter-spacing: normal;
+  color: var(--black);
+}
+
+.third {
+  margin: 0 40px 0 0;
+}
+
+.current {
+  opacity: 0.5;
+  margin: 1.5rem 0 0 0;
+  font-family: 'Roboto Slap', serif;
+  font-size: 0.813rem;
+  letter-spacing: normal;
+  color: #000;
+}
+
+.chapter {
+  margin: 5px 0 25px 0;
+  font-family: 'Roboto Slap', serif;
+  font-size: 1.3em;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #000;
+}
+
+.progress-btn {
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+}
+
+.progress {
+  font-family: 'Roboto Slap', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.form-container {
+  padding: 0 5em;
+}
+
+.form-title {
+  color: #888;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+  font-size: 1.5em;
+  letter-spacing: -0.18px;
+}
+
+#title {
+  width: 40vw;
+  height: 15vh;
+}
+
+#author {
+  width: 20vw;
+  height: 15vh;
+  margin: 0 3em;
+}
+
+.add-button {
+  width: 15vw;
+  height: 15vh;
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+  border: none;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  letter-spacing: 0.5px;
+  text-align: center;
+}
```

Comment:
- [ ] Please reduce the height for the form element (title, author, button)

Your design |
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/167270929-0c78f02d-9faa-40ce-8ece-f45e9ab09b02.png)

Figma Design |
:-----------------:|
![a2](https://user-images.githubusercontent.com/39523312/167270934-e85d5441-9086-4405-b40a-8842e49746e2.png)

Reply:
done